examples/istioctl-analyze/configs/example-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: httpbin-gateway
+spec:
+  selector:
+    istio: ingressgateway
+#   servers:
```

Comment:
?

Reply:
User just needs to uncomment this code to fix the error. I added a comment in the yaml to make it more clear.